src/main/src/test/java/org/geoserver/logging/LoggingStartupContextListenerTest.java

```diff
@@ -174,4 +175,36 @@ public void testFilenameAndFilePattern() throws Exception {
                 GeoServerXMLConfiguration.applyPathTemplate(
                         "${baseDir}/logs/logging.log", "logs/geoserver.log"));
     }
+
+    @Test
+    public void testNodeAttributes() {
+        Node node = new Node();
+        node.getAttributes().put("fileName", "logs/geoserver.log");
+
+        assertEquals("fileName", GeoServerXMLConfiguration.attribute(node, "fileName"));
+
+        assertEquals(
+                "attribute style",
+                "logs/geoserver.log",
+                GeoServerXMLConfiguration.attributeGet(node, "fileName"));
```

Comment:
Can I suggest adding a static import, and get more concise code?

Reply:
You can suggest indeed, not generally worried about concise code for test cases.

I am trying to keep my options open for moving this node pre-processing into a distinct class (so it can be shared if we every make a GeoServerYAMLConfiguration). Perhaps static import with package visibility is a good approach.